### ISSUE TITLE
Unnecessary <NuxtPage> in components.kun.Layout.vue

### ISSUE THREAD

**Issue by CloudeaSoft** [KUN1007/kun-galgame-nuxt3#28]
In [Layout.vue], it looks like this.
```
<template>
  <div class="app" :style="{ backgroundImage: `url(${imageURL})` }">
    <div class="top-bar">
      <KunTopBar />
    </div>
    <NuxtPage />
  </div>
</template>
```
But in [app.vue], the [Layout.vue] was used as a component with a slot tag

```
<template>
  <!-- ...... -->
  <KunLayout>
    <NuxtPage />
  </KunLayout>
</template>
```

**Comment by KUN1007** [comment]
Thanks for your feedback.
Your opinion is very correct, this is our mistake.

**Comment by KUN1007** [comment]
Seems like your solved this problem on #29 , closed.
